test/evptests.txt

```diff
@@ -2,6 +2,63 @@
 #aadcipher:key:iv:plaintext:ciphertext:aad:tag:0/1(decrypt/encrypt)
 #digest:::input:output
 
+# BLAKE2 tests, using same inputs as MD5
```

Comment:
They don't seem to match what's in the source, they all seem to also have a key which we don't have.  RFC7693 only seems to have 1 example for each.